go.mod

```diff
@@ -30,6 +30,7 @@ require (
 
 require (
 	github.com/confio/ics23/go v0.7.0
+	github.com/cosmos/ibc-go v1.2.2
```

Comment:
Check ibc-go versions.

Reply:
It's replaced to our ibc-go fork see
https://github.com/cosmos/interchain-security/blob/a8f854b777bc7c3bc0f1a2532dd084a8c10855fd/go.mod#L140